<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Images/360_controller.svg.png</normaloff>:/Images/360_controller.svg.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Main Window Resource.qrc">:/Images/360_controller.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Left Trigger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="l_trigger_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Right Trigger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="r_trigger_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Left Stick X Axis (left right)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="l_thumb_x_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Left Stick Y Axis (up down)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="l_thumb_y_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Steering Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="steering_style_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Speed Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="speed_style_combo"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Open|QDialogButtonBox::Save</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Main Window Resource.qrc"/>
 </resources>
 <connections/>
</ui>
